src/http/server.cr

```diff
@@ -138,8 +138,8 @@ class HTTP::Server
     end
   end
 
-  def bind
-    @server ||= TCPServer.new(@host, @port)
+  def bind(reuse_port = false)
```

Comment:
In the discussion of this issue we suggested to make it a property of `HTTP::Server`. What's wrong with that? That way it's used in both `bind` and `listen`, and if you stop the server and start it again the configuration is preserved.